pkg/solidity-utils/contracts/test/LinearBasePoolMathMock.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "../math/FixedPoint.sol";
+import "./BasePoolMathMock.sol";
+
+contract LinearBasePoolMathMock is BasePoolMathMock {
+    using FixedPoint for uint256;
+
+    function computeInvariant(uint256[] memory balances) public pure override returns (uint256) {
+        // inv = x + y
```

Comment:
Maybe add a comment that this is for our mock pools, which use linear math for simplicity.